src/shared/components/iframe-runtime.tsx

```diff
@@ -55,6 +58,12 @@ export const IframeRuntime: React.FC<IProps> =
       phone.addListener("hint", (newHint: IHintRequest) => {
         setHint(newHint.text || "");
```

Comment:
For the fullscreen interactive this should call setHint on the host. See my comment below where the hint is rendered.

Reply:
@eireland you might be busy with other stuff, but I just wanted to make sure you saw these comments.

Reply:
looks good now.